src/components/JobsTrainingContent/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import "./index.scss";
+import { useTranslation } from "react-i18next";
+import OpportunitiesCard from "../OpportunitiesCard";
+import SearchBar from "../SearchBar";
+
+const JobsTrainingContent = () => {
+  const { t } = useTranslation();
+  const [translationKey, setTranslationKey] = useState("jobOpportunities");
+  const cardsArr = [...t(`home.${translationKey}`, { returnObjects: true })];
+  const cards = cardsArr.map((opportunity) => (
+    <OpportunitiesCard {...opportunity} key={opportunity.id} />
+  ));
+
+  const cardsList = cards.map((card, index) => {
+    return (
+      <Col
+        sm={12}
+        md={6}
+        className={index % 2 === 0 ? "peachBorder" : "greenBorder"}
+        key={card.props.id}
+      >
+        {card}
+      </Col>
+    );
+  });
+
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  const selectButton = () => {
+    setIsDisabled(!isDisabled);
+  };
+
+  return (
+    <Container className="jobsTrainingContent">
+      <Row className="searchBar">
+        <SearchBar />
+      </Row>
+      <Row>
+        <Col md={3}>
+          <div>
+            <button
+              disabled={isDisabled}
+              type="button"
+              translationKey="button"
+              className="jobsBtn"
+              onClick={() => {
+                selectButton();
+                setTranslationKey("jobOpportunities");
```

Comment:
why didn't you move this to `selectButton` function? doing that would help you write this in one line like:
```
onClick={selectButton}
```

Reply:
ok I can change the name.